controllers/osbuildconfig_controller.go

```diff
@@ -30,7 +34,9 @@ import (
 // OSBuildConfigReconciler reconciles a OSBuildConfig object
 type OSBuildConfigReconciler struct {
 	client.Client
-	Scheme *runtime.Scheme
+	Scheme                  *runtime.Scheme
+	OSBuildConfigRepository osbuildconfig.Repository
```

Comment:
Don't you need to initialize these two with a client?

Reply:
Fixed it